src/main/java/org/apache/felix/scr/impl/ScrCommand.java

```diff
@@ -323,10 +323,10 @@ public int compare(Component c1, Component c2)
             {
                 out.print( "Services: " );
                 out.println( services[0] );
-                for ( int i = 1; i < services.length; i++ )
+                for ( String service: services )
```

Comment:
this changes the semantics. The first service is now printed twice?

Reply:
Yes, mistake in backport.
This is the original:
https://github.com/apache/felix/commit/b44672ee2f22ff616ebc47e686dfa4df696abbef
Will update